<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>split</class>
 <widget class="QWidget" name="split">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>split</string>
  </property>
  <widget class="QStackedWidget" name="split_pages">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>760</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>850</width>
     <height>600</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="split_openfile_page">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>2</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>760</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QPushButton" name="split_open_file_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>550</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>760</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>850</width>
       <height>600</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="split_para_page">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>2</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>760</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QPushButton" name="split_output_path_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择输出文件夹</string>
     </property>
    </widget>
    <widget class="QLabel" name="split_end_row_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>结束行号: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="split_end_row_entry">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>250</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="split_input_rows_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入要拆分的起始行号和结束行号(1-1000000):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="split_start_row_entry">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="split_select_file_path">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 247, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="split_start_row_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起始行号: </string>
     </property>
    </widget>
    <widget class="QLabel" name="split_select_file_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>已选择文件:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="split_column_entry">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="split_process_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始处理</string>
     </property>
    </widget>
    <widget class="QLabel" name="split_column_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请问根据哪一列的内容进行拆分(例如:1或A):</string>
     </property>
    </widget>
    <widget class="QLabel" name="split_output_path_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 247, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
